src/pages/Staking/index.tsx

```diff
@@ -241,12 +258,13 @@ const Staking = (): JSX.Element => {
       }
       const unlockHeight = currentBlockNumber + convertWeeksToBlockNumbers(variables.time);
 
-      return window.bridge.interBtcApi.escrow.createLock(variables.amount, unlockHeight);
+      return window.bridge.escrow.createLock(variables.amount, unlockHeight);
     },
     {
       onSuccess: () => {
         voteGovernanceTokenBalanceRefetch();
         stakedAmountAndEndBlockRefetch();
+        currentRewardAmountRefetch();
```

Comment:
@nud3l 
I don't think we should call this because I think claimable reward amount is supposed to be increased per block so as soon as staking, this information will not be updated.

Reply:
The claimable amounts are going to be changed when you change the stake amount or locktime

Reply:
This kind of refetch, technically we call it invalidation in react-query terms, is to be used to reflect the changes made on the backend side when some other relevant action is taken.
But here as soon as the user has staked or added more stake, I don't think this claimable amount will be changed on the parachain side because as far as I know, it is supposed to be calculated per block.

Although `currentRewardAmountRefetch` is called, visually the claimable amount stays the same at that moment of `currentRewardAmountRefetch` being called. I tested it and confirmed. So `currentRewardAmountRefetch` is called for no effective reason I'd say. Please correct me if I'm wrong.